### PR TITLE
Register Subdomain: quydang

### PR DESCRIPTION
**Pull request by quydang04**
Added the domain: quydang

### CODE REVIEW

**Comment by quydang04** [comment]
Thanks you very much

File: domains/quydang.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "quydang04",
+        "email": "dangvonguyen169@gmail.com"
+      },
+      "record": {
+        "CNAME": "github.io"
```
Comment (WilliamDavidHarrison) [inline domains/quydang.json:8]:
```suggestion
        "CNAME": "quydang04.github.io"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]
If you want to use GitHub Pages, commit the suggestion below.

**Comment by quydang04** [comment]
> If you want to use GitHub Pages, commit the suggestion below.

I have just updated, please check for me. Thank you very much.

**Review by wdhdev — CHANGES_REQUESTED** [review]
Sorry you need to create a website

**Review by quydang04 — COMMENTED** [review]
Yes, i have fixed this. Check for me again, please.

**Review by wdhdev — APPROVED** [review]

**Pull request merged**